README.md

```diff
@@ -132,6 +132,19 @@ Your last Laptop run will be saved to `~/laptop.log`. Read through it to see if
 you can debug the issue yourself. If not, copy and paste the entire log into a
 [new GitHub Issue](https://github.com/18F/laptop/issues/new) for us.
 
+#### Git Seekrets Troubleshooting
+
+If you're running into issues after installing `git-seekret`, make sure that you
+install the necessary dependencies.
+
+```shell
+brew install openssl
```

Comment:
This assumes that Homebrew is already installed. I would suggest either:
1) Add instructions for installing Homebrew
or 
2) Require running the laptop script for now until we implement a more advanced à la carte menu where you can pick and choose what gets installed